pkg/cvo/updatepayload.go

```diff
@@ -42,10 +42,16 @@ const (
 	imageReferencesFile = "image-references"
 )
 
-func loadUpdatePayload(dir, releaseImage string) (*updatePayload, error) {
+type payloadTasks struct {
+	idir       string
+	preprocess func([]byte) ([]byte, error)
+	skipFiles  sets.String
+}
+
+func loadUpdatePayloadMetadata(dir, releaseImage string) (*updatePayload, []payloadTasks, error) {
```

Comment:
any reason why `loadUpdatePayloadMetadata` returns the payload tasks?

Reply:
Right now the CVO doesn't know what version its payload has.  I wanted to share the same code for reading the default payload in the operator init path (where it tries to read the current payload to find its current version) and in when we apply it, so I extracted the first chunk out.  payload tasks encapsulated all the path specific stuff, so it avoided the remainder of loadUpdatePayload from knowing anything about paths.